modular/gater/errors.go

```diff
@@ -54,9 +54,9 @@ var (
 	ErrReplyData              = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50037, "reply the downloaded data to client failed")
 	ErrTaskMsgExpired         = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50038, "the update time of the task has exceed the expire time")
 	ErrSecondaryMismatch      = gfsperrors.Register(module.GateModularName, http.StatusNotAcceptable, 50039, "secondary sp mismatch")
-	ErrPrimaryMismatch        = gfsperrors.Register(module.GateModularName, http.StatusNotAcceptable, 50040, "primary sp mismatch")
-	ErrNotCreatedState        = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50041, "object has not been created state")
-	ErrNotSealedState         = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50041, "object has not been sealed state")
+	ErrPrimaryMismatch        = gfsperrors.Register(module.GateModularName, http.StatusNotAcceptable, 50041, "primary sp mismatch")
```

Comment:
Why skips `50040` error code?

Reply:
50040 already used
` ErrSignedMsgNotMatchSPNonce  = gfsperrors.Register(module.GateModularName, http.StatusBadRequest, 50040, "The SP Nonce in the signed message in "+GnfdAuthorizationHeader+" is not for the this SP.")`